concepts/ofd/index.md

```diff
@@ -0,0 +1,372 @@
+
+If a validator has been jailed by AFD then all staking rewards earned by the validator in the epoch are _forfeited_ and distributed to the _reporting validator_ as described in [Accountability fault Detection (AFD), Slashing penalties](/concepts/accountability/#slashing-penalties).
+
+If a validator has been jailed by OFD then all staking rewards earned by the validator in the epoch are _wittheld_ and distributed to the _withheld rewards pool account_ for community funding as described in [Slashing penalties](/concepts/ofd/#slashing-penalties) on this page.
+
+:::
+
+## Protocol configuration
+
+Slashing penalties for an _offending validator_ are computed by the protocol based on the offending validator's _aggregated inactivity score_, its _offence_ history, if it is on _probation_, and the total number of offences committed in the current and preceding epoch by other committee members, i.e. a _collusion degree_. For the omission fault parameters see [slashing protocol configuration](/concepts/ofd/#slashing-protocol-configuration) beneath.
+
+Applied slashing penalties cover withholding of ATN staking rewards and NTN Newton inflation rewards proportional to the % of offline blocks in the epoch, probation, jailing, and slashing a percentage of stake. Slashing and probation scale up quadratically by the number of repeated offences.
+
+Penalties are applied for proven omission faults at epoch end. 
+
+Slashing is applied as part of the state finalization function. As the last block of an epoch is finalized, OFD will apply slashing for proven _omission faults_ to withhold ATN staking rewards and NTN inflation rewards, slash validator stake per Autonity's [Penalty-Absorbing Stake (PAS)](/glossary/#penalty-absorbing-stake-pas) model, and applying validator jailing if detected committing an _omission fault_ while on _probation_.
+
+### Protocol parameters
+
+OFD protocol parameters are set by default to:
+
+| Protocol parameter | Description | Value |
+|:--:|:--|:--:|
+| _inactivity threshold_ | the threshold to determine if a committee member is an _offending validator_ at the end of the epoch | `1000` (10%) |
+| _lookback window_ | the number of blocks over which the protocol will look for inactivity | `40` (40 blocks) |
+| _past performance weight_ | a factor that determines how much weight is given to past performance of the validator in the preceding epoch. The factor is subtracted from the validator's performance in the current epoch when computing the aggregated inactivity score | `1000` (10%) |
+| _initial jailing period_ | the initial number of epoch(s) that an _offending validator_ will be jailed for | `10_000` (10000 blocks) |
+| _initial probation period_ | the initial number of epoch(s) that an _offending validator_ will be set under probation for | `24` (24 epochs) |
+| _initial slashing rate_ | the division precision used as the denominator when computing the slashing amount of a penalty | `25` (0.25%) |
```

Comment:
this definition needs to be corrected

Reply:
Corrected as done for comment https://github.com/autonity/docs.autonity.org/pull/259#discussion_r2063589147.  Commit 200393b55d70094556206c67eb6661a40614de87